### ISSUE TITLE
Can't shutdown sys-net, killing it freezes the system, upon restart the OS doesn't load.

### ISSUE THREAD

**Issue by NinjaTurtle007** [QubesOS/qubes-issues#3342]
#### Qubes OS version:

4 RC3

#### Affected TemplateVMs: 

sys-net

---

### Steps to reproduce the behavior:

1. shutdown sys-net from the top right applet
2. kill it (I waited for at least 5 mins before killing it)
3. system freezes
4. restart
5. it hangs on starting sys-net for over half hour (I didnt wait anymore)

### Expected behavior:

To just shutdown when told so from the applet

### Actual behavior:

It hangs on shutdown

### General notes:
Basically for me, simple attempt to shutdown sys-net led to reinstalling the whole OS

---
